import_obfuscation.go

```diff
@@ -87,6 +148,9 @@ func obfuscateImports(objPath, tempDir, importCfgPath string) (garbledObj string
 	if err != nil {
 		return "", nil, nil, fmt.Errorf("error parsing main objfile: %v", err)
 	}
+	if err := extractDebugObfSrc("main", mainPkg); err != nil {
```

Comment:
"main" is the main package's package name, but not its import path. For example, garble's main package has the import path mvdan.cc/garble.

Reply:
okgPath != import, "main" is the name of the folder where the obfuscated sources should be unpacked.

https://github.com/burrowers/garble/pull/158/files/3889fcdaab8b73ee6167d6b2eb81ac24792454c2#diff-449f0864b56725fec5bfd968484b654c8f71a67423a71e839bca894402ecb4bdR173
Here comes an additional check that the main object file is not extracted twice.

Reply:
Hmm, I'm pretty sure that the terms "package path" and "package import path" are equivalent. But if you're sure that this code behaves the same as master, I guess there's no harm in it.

I really do want to revisit this, though, because most of the occurrences of `"main"` are probably bugs.

Reply:
I copied the @capnspacehook approach, hopefully it can tell me the right approach or not.